oak-upgrade/src/main/java/org/apache/jackrabbit/oak/upgrade/cli/parser/OptionParserFactory.java

```diff
@@ -174,5 +176,6 @@ private static void addMiscOptions(OptionParser op) {
         op.accepts(SKIP_CHECKPOINTS, "Don't copy checkpoints on the full segment->segment migration");
         op.accepts(FORCE_CHECKPOINTS, "Copy checkpoints even if the --include,exclude,merge-paths option is specified");
         op.accepts(ADD_SECONDARY_METADATA, "Adds the metadata required by secondary store");
+        op.accepts(SKIP_CREATE_AZURE_CONTAINER, "Create azure container if not exist");
```

Comment:
Text should be the other way around: "Skip creating the Azure container if it does not exist".